src/test/java/com/google/api/codegen/gapic/testdata/java/java_library.baseline

```diff
@@ -119,4874 +119,4879 @@ clean {
   delete 'all-jars'
 }
 
-============== file: src/main/java/com/google/example/library/v1/LibraryClient.java ==============
-/*
- * Copyright 2019 Google LLC
- *
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- *     https://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
-package com.google.example.library.v1;
+============== file: samples/src/main/java/com/google/example/examples/library/v1/snippets/DiscussBookCallableCallableStreamingBidiProg.java ==============
```

Comment:
Can we remove `snippets/` from the path? It doesn't add any semantic/organizational value, AFAICT.

Reply:
Also, I would suggest s/`examples`/`samples`/. Is that path element only used in Java?
(If it's simple to implement both of these suggestions, let's do them now. But if it complicates the PR too much, I'm fine doing this in a chaser PR.)